lib/SourceMapDevToolPlugin.js

```diff
@@ -145,7 +146,10 @@ class SourceMapDevToolPlugin {
 							query,
 							basename: basename(filename)
 						});
-						const sourceMapUrl = path.relative(path.dirname(file), sourceMapFile).replace(/\\/g, "/");
+						let sourceMapUrl = path.relative(path.dirname(file), sourceMapFile).replace(/\\/g, "/");
+						if(sourceMapUrl.indexOf("[contenthash]") !== -1) {
+							sourceMapUrl = sourceMapUrl.replace("[contenthash]", crypto.createHash("md5").update(source).digest("hex"));
```

Comment:
Use a regexp instead to replace every occurrence.

`replace(/\[contenthash\]/g, `